flocker/ca/functional/test_ca.py

```diff
@@ -133,6 +133,21 @@ def test_node_certificate(self):
         )
 
     @requireCA
+    def test_apiuser_certificate(self):
+        """
+        Test for ``flocker-ca create-api-certificate`` command.
+        Runs ``flocker-ca initialize`` followed by
+        ``flocker-ca create-api-certificate` and calls ``openssl``
+        to verify the generated control certificate and private key is
+        signed by the previously generated certificate authority.
+        """
+        flocker_ca("initialize", "mycluster")
```

Comment:
`flocker_ca` uses `check_output` which leaks bytes onto the parents stdio streams.  It would be better if it used `flocker.testtools.run_process`.